core/src/display_object/movie_clip.rs

```diff
@@ -4611,6 +4611,7 @@ struct MovieClipStatic<'gc> {
     /// The last known symbol name under which this movie clip was exported.
     /// Used for looking up constructors registered with `Object.registerClass`.
     exported_name: GcCell<'gc, Option<AvmString<'gc>>>,
+    avm2_class: GcCell<'gc, Option<Avm2ClassObject<'gc>>>,
```

Comment:
This will be the fourth GcCell in `MovieClipStatic`. And AFAIK there's also a preload step that copies the `MovieClipStatic` to change some of the other fields. As we find all these partial/delayed initialization cases I'm starting to wonder when it would make sense to make this mutable and call it `MovieClipShared` or `MovieClipDefinition` instead...

Reply:
That sounds reasonable - I can do that in a follow-up pr